nilearn/regions/tests/test_parcellations.py

```diff
@@ -109,18 +109,6 @@ def test_parcellations_fit_on_multi_nifti_images(method, test_image):
 
     assert parcellator.labels_img_ is not None
 
-    parcellator = Parcellations(method="rena", n_parcels=5, verbose=0)
-    parcellator.fit(fmri_imgs)
-
-    assert parcellator.labels_img_ is not None
-
-    parcellator = Parcellations(
-        method="hierarchical_kmeans", n_parcels=5, verbose=0
-    )
-    parcellator.fit(fmri_imgs)
-
-    assert parcellator.labels_img_ is not None
-
```

Comment:
both rena and hierarchical_kmeans were actually included in the parametrization of this test